### PR TITLE
Clean up CardRenderer.

### PR DESCRIPTION
**Pull request by kberg**
There was a lot of pop and push when it wasn't necessary in CardRenderer, making the class tricky to read. That and a little cleanup makes for a completely identical output.

I know this for two reasons: the first is that I visually inspected every card, and they matched on the `/cards` path. The second is that I extracted the full card renderings, and ran diffs until they were exactly the same:

image

I've done a fair bit of analysis of the non-method elements of the rendering classes, and I am confident that the export is the complete set.

### CODE REVIEW

**Comment by kberg** [comment]
Actually I need to run `npm run test` one more time. That was a pretty confident statement to make before running the tests, I hope I don't eat crow! :D

File: src/cards/render/CardRenderer.ts [hunk 1]
```diff
@@ -704,67 +577,50 @@ class Builder {
...
+    const row = this._currentRow();
+    const item = row.pop();
+    if (!(item instanceof CardRenderItem)) {
+      throw new Error('"brackets" could be called on CardRenderItem only');
+    }
+
+    if (item === undefined) {
+      throw new Error('Called "brackets" without a CardRenderItem.');
     }
+    row.push(CardRenderSymbol.bracketOpen());
```
Comment (bafolts) [inline src/cards/render/CardRenderer.ts:621]: The `push` function takes multiple arguments. This could be written:

```
row.push(CardRenderSymbol.bracketOpen(), item, CardRenderSymbol.bracketClose());
```

Noticed this while reviewing and realize this is unrelated to this change. Should be slightly more efficient.

**Review by bafolts — APPROVED** [review]

File: src/cards/render/CardRenderer.ts (hunk 1 above)
Comment (kberg) [reply src/cards/render/CardRenderer.ts:621]: ack. That's fair.
